From 325534f013c9b1a46d7587048f913a1984c46802 Mon Sep 17 00:00:00 2001
From: Mika Havela <mika.havela@gmail.com>
Date: Wed, 9 Feb 2011 10:21:15 +0100
Subject: [PATCH] Bugfix: If hostname -f does not work, then ssl creation will break and acf won't work

FQDN hostname might not be set in /etc/hosts and this could cause 'hostname -f' to fail. Even this does not happen too often, it still is a possibility. This patch prevents the problem.
---
 setup-acf.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup-acf.in b/setup-acf.in
index 4f2afec..c6f621e 100644
--- a/setup-acf.in
+++ b/setup-acf.in
@@ -75,7 +75,7 @@ prompt = no
 
 [ req_dn ]
 OU=HTTPS server
-CN=$(hostname -f)
+CN=$(hostname -f || hostname)
 emailAddress=${EMAIL:-postmaster@example.com}
 
 [ cert_type ]
-- 
1.7.3.5

